contents/uix/accessibility.md

```diff
@@ -0,0 +1,181 @@
+
+Such disabilities might affect website usage. It could be due to difficulty in understanding how to complete a task, remembering how to do something that was previously accomplished, increased frustration at confusing workflows or inconsistent layouts/navigation/other page features.
+
+Unlike other web accessibility issues, there is no quick fix to issues arising from cognitive disabilities. The rule of thumb you can follow is to always design your websites to be as logical, consistent, and usable as possible. Here is a checklist that you can follow:
+
+* **Avoid distractions**: <br>
+People with cognitive disabilities are more easily distracted, or they may have heightened sensory awareness where cluttered pages can be overwhelming.
+  * Avoid unnecessary movements from animations or moving elements
+  * Have a clean and uncluttered design
+  
+  Here's a comparison of a cluttered vs clean website landing page taken from British Gas website.
+  
+  Cluttered page:
+  ![Cluttered landing page](accessibility/bg-old.png "Cluttered landing page")
+
+  Clean page:
+  ![Clean landing page](accessibility/bg-new.png "Clean landing page")
+
+* **Be logical**:
+  * Multi-stage processes are broken down into logical steps, with regular reminders of how far through the process you are, and how long you've got left to complete the process, if appropriate.
+  * Workflows are logical, simple, and require as few interactions as possible to complete.
+
+* **Be consistent**:
+  * Navigation, header, footer, and main content are always in the same places
+  * Navigation mechanisms are consistent in appearance and behaviour
+
+* **Communicate well**: <br>
+People with cognitive disabilities are more likely to take things literally and can benefit from more visual cues.
+  * Pages are not overly long or dense in terms of the amount of information presented at once.
+  > [An article](https://www.lifewire.com/web-page-length-3468959) says a rule of thumb is "No article should exceed 2 printed pages of double-spaced, 12 point text." whereas [another article](https://www.greenmellenmedia.com/how-long-should-my-website-content-be/) says "There is no "magic number" for a number of characters for content. As long as the web page content is clear, to the point and written for people". Ultimately, what's most important is what _your_ users think, so validation during user testing is paramount.
+  * The language used is as plain and easy to follow, and not full of unneccessary jargon and slang.
+  * Important points and content are highlighted in some way.
+  * User errors are clearly highlighted, with help messages to suggest solutions.
+
+  <blockquote>
```

Comment:
Looks a little confusing:
![image](https://user-images.githubusercontent.com/25320782/54522870-3cced780-49a9-11e9-8331-fe33794a37b6.png)

Not sure if there is a better way to present this.